Packages/com.unity.inputsystem/InputSystem/Plugins/XR/TrackedPoseDriver.cs

```diff
@@ -1,5 +1,6 @@
 using System;
 using UnityEngine.InputSystem.LowLevel;
+using UnityEngine.XR;
```

Comment:
as @chris-massie mentioned, need to add a dependency on `com.unity.modules.xr` for this.

Reply:
(in package.json, just to be clear)